framework/utils/InputData.py

```diff
@@ -113,6 +113,26 @@ def convert(cls, value):
 
 IntegerType.createClass("integer","xsd:integer")
 
+#
+#
+#
+#
+class BooleanType(InputType):
```

Comment:
BoolType is available in InputData, why do you need to define a new method?

Reply:
Ah, it was buried beneath all the list types. We should try to maintain some sort of order. Removed.